__tests__/app.test.js

```diff
@@ -81,3 +81,55 @@ describe("GET /api/articles/:article_id", () => {
     return request(app).get("/api/apple").expect(404);
   });
 });
+describe("GET /api/articles", () => {
+  it("200: should return an array", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(Array.isArray(articles)).toBe(true);
+      });
+  });
+  it("200: responds with an array of all topics", () => {
```

Comment:
You could merge all three of these tests into one to prevent having to make the same request over and over again. That being said I really like the is array and length assertions before looping over to check the properies and values